public/locales/auset-fr.json

```diff
@@ -0,0 +1,426 @@
+      "setupTitle": "Configurer le Complot corvidé",
+      "setup": "<i>1.</i> Choisissez une clairière de la patrie. Placez-y 1 <strong>guerrier</strong> et 1 <strong>jeton complot</strong> de votre choix.<br/><i>2.</i> Placez 1 <strong>guerrier</strong> dans une clairière de chaque couleur. <i>(En comptant l'étape précédente, placez 4 guerriers au total.)</i>"
+    },
+    "cult": {
+      "name": "Culte du lézard",
+      "summaryTitle": "Jouer le Culte du lézard",
+      "summary": "<p>En tant que Culte des Lézards, vous vous occupez des créatures qui ont été rejetées par les autres factions. Vous marquez des points de victoire en effectuant les <strong>rituels</strong> appropriés, en révélant les cartes de votre main correspondant aux clairières où vous avez <strong>jardins</strong>, et défausser des cartes pour terminer le rite. Plus vous avez de jardins dans les clairières correspondant à la carte révélée, plus vous marquez de points.</p><p>Votre approche plus douce signifie que vous ne pouvez pas combattre votre ennemis au début; pour ce faire, vous devez radicaliser vos partisans en <strong>acolytes</strong>. Ce faisant, vous pouvez effectuer des <strong>conspirations</strong> dans les clairières du <strong>Paria</strong> , la couleur la plus défaussée au cours du tour précédent.</p><p>Votre <strong>Haine des oiseaux</strong> signifie que vos cartes oiseaux ne sont pas sauvages dans vos rituels. Vos jardins répandent la nouvelle de votre seigneur dragon, attirant des foules de <strong>Pèlerins</strong>, vous dirigez donc une clairière si vous n'y avez qu'un seul jardin. Enfin, vos acolytes obtiennent toujours <st rong>Vengeance</strong> sur les méchants: chaque fois qu'un de vos guerriers est éliminé alors qu'il défendait, vous gagnez un autre acolyte.</p>",
```

Comment:
Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.